exec/selected/selected.go

```diff
@@ -55,6 +55,60 @@ type SchemaField struct {
 	FixedResult reflect.Value
 }
 
+func (sf *SchemaField) ToSelectedField() *types.SelectedField {
```

Comment:
i only dont like that you recompute this value every single time you access fields

in requested-fields this was precomputed and then accessed

ideally we should do it the same way, but it's fine for our use cases for now

ticket it

Reply:
AMP-1615